src/Goerli-DssSpell.t.sol

```diff
@@ -1794,6 +1794,7 @@ contract DssSpellTest is DSTest, DSMath {
                 assertEq(abi.decode(val, (address)), address(dog), string(abi.encodePacked("TestError/reg-xlip-dog-", ilk)));
                 }
                 {
+
```

Comment:
nit: extra line

Reply:
lol, fixed but you have to deal with my commit message.